docker/pythonpath_dev/superset_config.py

```diff
@@ -224,6 +231,9 @@ class CeleryConfig:
         }
     }
 
+if DOMAIN_WHITELIST:
+    AZURE_PROVIDER['whitelist'] = DOMAIN_WHITELIST
```

Comment:
What is the difference compared to line 216?

Reply:
that is a bug, good catch!